src/cascadia/QueryExtension/GithubCopilotLLMProvider.cpp

```diff
@@ -194,7 +194,9 @@ namespace winrt::Microsoft::Terminal::Query::Extension::implementation
         }
         catch (...)
         {
-            _AuthChangedHandlers(*this, winrt::make<GithubCopilotAuthenticationResult>(RS_(L"UnknownErrorMessage"), nullptr));
+            // some unknown error happened and we didn't get an "error" key, bubble the raw string of the last response if we have one
```

Comment:
won't this result in it printing a LLM message two times if it gets an unknown error?

Reply:
We won't end up in this `catch` block if the json was parsed correctly (if we saw an error in the json block then it still means we parsed it correctly and we just raise that one), so I believe we only raise this once unless I'm missing something

Reply:
Sorry, what I mean is

Say the user asks the LLM a question and gets a reply, "Hello I am a computer"

We save that as the last response.

Next, they ask a question and get an unparseable error.

Are we going to load up the last response ("Hello I am a computer") and display that _again_?

Since we "bubble the raw string of the last response".